README.md

```
@@ -1,16 +1,38 @@
 # bdd_demo
 
-A new Flutter project.
+Demo BDD ideas with patrol
 
 ## Getting Started
 
-This project is a starting point for a Flutter application.
+Using
+https://plugins.jetbrains.com/plugin/9164-gherkin
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)